tc_hivemind_backend/ingest_qdrant.py

```
@@ -47,21 +50,38 @@ def __init__(
 
         credentials = Credentials()
-        _, self.embedding_dim = load_model_hyperparams()
-        self.pg_creds = credentials.load_postgres()
+        if embed_model is None:
+            if embed_dim is None:
+                _, self.embedding_dim = load_model_hyperparams()
+            else:
+                self.embedding_dim = embed_dim
+        else:
+            if embed_dim is None:
+                raise ValueError("Embed dimension should be provided if embed model is provided!")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)